app/src/main/java/org/mozilla/fenix/components/accounts/FenixAccountManager.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.accounts
+
+import android.content.Context
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import org.mozilla.fenix.ext.components
+
+/**
+ * Component which holds a reference to [FxaAccountManager]. Manages account authentication,
+ * profiles, and profile state observers.
+ */
+open class FenixAccountManager(context: Context) {
+    val accountManager = context.components.backgroundServices.accountManager
+    val authenticatedAccount = accountManager.authenticatedAccount() != null
```

Comment:
This should be a function, as it's now it is only going to store the first value when the object of `FenixAccountManager` is created, instead of constantly querying the latest value from `accountManager.authenticatedAccount()`.

Reply:
You could use easily solve this by using  `get() =`